pkg/apis/migration/v1alpha1/v1alpha1_suite_test.go

```diff
@@ -40,16 +42,24 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		golog.Fatal(err)
 	}
-
-	if cfg, err = t.Start(); err != nil {
-		golog.Fatal(err)
+	setupControlPlane := os.Getenv("SETUP_CONTROL_PLANE")
```

Comment:
To be precise, this can be used in non-MacOS testing as well. The fact that control plane can take some time to come up and in my case I was never able to bring it up successfully, restricted me with ability to run unit test that dont depend on control plane